cattle-config-imp.yaml

```diff
@@ -0,0 +1,70 @@
+aksClusterConfig:
```

Comment:
**nit:**
Can you rename this file to `cattle-config-import.yaml` and the other one to `cattle-config-provisionin.yaml`?
`imp` always reminds me of `important` and I always get confused between the 2 files.